docs/design/2021-04-26-lock-view.md

```diff
@@ -88,8 +105,9 @@ Several tables will be provided in `information_schema`. Some tables has both lo
 | `RETRYABLE` | `bool` | Is the deadlock retryable. TiDB tries to determine if the current statement is (indirectly) waiting for a lock locked by the current statement. |
 | `TRY_LOCK_TRX_ID` | `unsigned bigint` | The transaction ID (start ts) of the transaction that's trying to acquire the lock |
 | `CURRENT_SQL_DIGEST` | `text` | The SQL that's being blocked |
+| `SQL_TEXT` | `text` | The SQL text of `CURRENT_SQL_DIGEST` |
+| `KEY_INFO` | `text` | Information of the key |
 | `KEY` | `varchar` | The key that's being locked, but locked by another transaction in the deadlock event |
-| `ALL_SQL_DIGESTS` | `text` | A list of the digest of SQL statements that the transaction has executed |
```

Comment:
Is it certain that we don't want this field or at least won't implement it as a field here?

Reply:
At least I'm not thinking about implementing it in this sprint. You can remove it for now.
If we add it in the future, I think it won't be a breaking change.